### ISSUE TITLE
Support Non-Veteran Claimants

### ISSUE THREAD

**Issue by geoffreystienblock** [department-of-veterans-affairs/caseflow#11997]
## Background/context
This epic covers supporting non-veteran claimants for OAR and for the Board of Veterans' Appeals. Any other business line support will be scoped in a separate ticket.

### OAR
As a Claims Assistant I want to be able to process appeals for claimants who may not be Veterans so that we can process and move appeals forward for people who may be designees of the Veteran and the appeal still moves forward.

Caseflow users need the ability to process appeals for claimants who are not Veterans.  In VBMS, the Payee Code for Veterans is 00.  There are numerous other payee codes that cover spouses, children, third-parties, etc.  These need to be supported in Caseflow so AMA claims for non-Veterans can be established in Caseflow and processed in VBMS.

### The Board
As an Intake user, I need to be able to intake appeals where the appellant is not in BGS (CorpDB).

## Goals
- VBA Intake users can process non-veteran claimants not currently available in Caseflow
- Board Intake users can process non-Veteran appellants not currently available in Caseflow

## Stakeholders - OAR 
- Jane Chi
- Lauren Sylvia
- Fred Somers

## Stakeholders - BVA 
- John Gosnell
- Mike Sanford

**BVA SMEs**
- Mail Team: Kendra Jones, Ivy Wilson
- Clerk of the Board: Donnie Hachey (Donnie.Hachey@va.gov), Sara Cooney ( Sara.Delhauer@va.gov)

### Board
- _Draft: As a BVA intake user, I need a way to add a non-Veteran claimant even if the Veteran has no known relationships or is deceased_
- [As a BVA Caseflow Intake user, I should be able to add non-veteran claimants that don't exist in the Corporate Database](https://github.com/department-of-veterans-affairs/caseflow/issues/13964)
- _Draft: As a BVA Intake user, I need a way to add PoA information so that required correspondence can be sent out_
- _Draft: As a BVA Intake user, I need a way to review information I add for non-Veteran claimants so I can ensure I am entering the information as accurately as possible to prevent the need for future users to have to edit this_
- _Draft: As a BVA Intake user, I need a way to view added claimant information I added so that I can complete intaking the appeal_
- _Draft: As a BVA user (specific user TBD), I need a way to add and assign a task to VLJ Support (TBD) add a claimant to an external database_
- _Draft: As a Caseflow Intake user, I should be able to view claimant data that was manually added for board appeals for claimants not listed in the corporate database_
- _Draft: As a BVA user (specific user TBD), I need a way to quickly identify an appeal with a non-Veteran claimant so I can address it appropriately_
- [As a BVA Caseflow Intake user, I should be able to edit claimant data that was manually added for Board appeals for claimants not listed in the Corporate Database, in case there was a mistake](https://app.zenhub.com/workspaces/foxtrot-5f075a9e28abd0001de67d7b/issues/department-of-veterans-affairs/caseflow/13965)

## Open Questions/Use Cases (BVA)
- What will happen if an attorneys address doesn’t match between the 10182 form and the Corp DB? How can this be intaken? 
- If an attorney is in the CorpDB but their address is incorrect, what should the business process be to update this? In which system?
- If there are multiple claimants (e.g. father-son attorneys) with the same name but different addresses/DOB etc, how does an Intake user know if they're selecting the correct one?

### Out of Scope
Special functionality for business lines outside of VBA and the Board.

## References/Resources

[Product Brief](https://docs.google.com/document/d/1K2dPq7Zclb3M2FvdfoxHpQWZptHvcNxIc6zDCfDiIYg/edit)
[Project Timeline & Index](https://docs.google.com/document/d/1-ED4fMCM1s5S3UGM3LzyuPd4JB2aSPugO5XNDPT9wQw/edit#)

### Designs and workflows
- Design Prototype WIP
- Discovery Synthesis
- Story - 

## Non-Veteran Payee Codes

VBMS Payee Codes
> Submitted by AMO

Payee | Code | Payee | Code | Payee | Code
-- | -- | -- | -- | -- | --
Veteran | 00 | C&P Tenth Child | 20 | CH35 First Child | 41
First Payee Recipient | 01 | C&P Eleventh Child | 21 | CH35 Second Child | 42
Second Payee Recipient | 02 | C&P Twelfth Child | 22 | CH35 Third Child | 43
Third Payee Recipient | 03 | C&P Thirteenth Child | 23 | CH35 Fourth Child | 44
Fourth Payee Recipient | 04 | C&P Fourteenth Child | 24 | CH35 Fifth Child | 45
Fifth Payee Recipient | 05 | C&P Fifteenth Child | 25 | CH35 Sixth Child | 46
Sixth Payee Recipient | 06 | C&P Sixteenth Child | 26 | CH35 Seventh Child | 47
Seventh Payee Recipient | 07 | C&P Seventeenth Child | 27 | CH35 Eighth Child | 48
Eighth Payee Recipient | 08 | C&P Eighteenth Child | 28 | CH35 Ninth Child | 49
Ninth Payee Recipient | 09 | C&P Nineteenth Child | 29 | Dependent Father | 50
Spouse | 10 | Vendor | 30 | Dependent Mother | 60
C&P First Child | 11 | Consolidated Payee 1st group of children | 31 | DIC Award 38 USC412(a) First Payee | 70
C&P Second Child | 12 | Consolidated Payee 2nd group of children | 32 | DIC Award 38 USC412(a) Second Payee | 71
C&P Third Child | 13 | Consolidated Payee 3rd group of children | 33 | DIC Award 38 USC412(a) Third Payee | 72
C&P Fourth Child | 14 | Consolidated Payee 4th group of children | 34 | DIC Award 38 USC412(a) Fourth Payee | 73
C&P Fifth Child | 15 | Consolidated Payee 5th group of children | 35 | DIC Award 38 USC412(a) Fifth Payee | 74
C&P Sixth Child | 16 | Consolidated Payee 6th group of children | 36 | DIC Award 38 USC412(a) Sixth Payee | 75
C&P Seventh Child | 17 | Consolidated Payee 7th group of children | 37 | DIC Award 38 USC412(a) Seventh Payee | 76
C&P Eighth Child | 18 | Consolidated Payee 8th group of children | 38 | DIC Award 38 USC412(a) Eighth Payee | 77
C&P Ninth Child | 19 | Consolidated Payee 9th group of children | 39 | DIC Award 38 USC412(a) Ninth Payee | 78
First Claimant Burial/Accrued | 80 | Seventh Claimant Burial/Accrued  | 86
Second Claimant Burial/Accrued | 81| Eighth Claimant Burial/Accrued | 87
Third Claimant Burial/Accrued | 82 | Ninth Claimant Burial/Accrued | 88
Fourth Claimant Burial/Accrued | 83 | Tenth Claimant Burial/Accrued | 89
Fifth Claimant Burial/Accrued | 84 | Institutional Veteran CFR3.852 | 99
Sixth Claimant Burial/Accrued | 85 | |

---------------------------------------------------------

**Title:**  Caseflow supports establishment of Non Payee 00 claims

**Value Statement:**  As a Caseflow user, I need the ability to establish AMA claim claims in Caseflow for any payee code as defined in Supporting Artifact 65147, so that I am able to establish additional types of AMA claims such as apportionments and death claims.

**Definition of Done:**  Caseflow users need the ability to process appeals for claimants who are not Veterans.  In VBMS, the Payee Code for Veterans is 00.  There are numerous other payee codes that cover spouses, children, third-parties, etc.  These need to be supported in Caseflow so AMA claims for non-Veterans can be established in Caseflow and processed in VBMS.

**Acceptance Criteria:**  Users are able to establish AMA claims in Caseflow for all payee types identified in Supporting Artifact 65147.

**Comment by leikkisa** [comment]
I'm posting all of the current payee codes supported, and which cases they are supported for.

On higher level reviews and supplemental claims where the benefit type is either compensation or pension, we currently use payee_code "00" for the veteran. If the claimant is not the veteran, the user can select from the following list of payee codes at their discretion.

When a claim is automatically generated (for remands and DTAs), the payee code is selected based on the relationship type or data from past claims.

```
  '01': '01 - First Payee Recipient',
  '02': '02 - Second Payee Recipient',
  '03': '03 - Third Payee Recipient',
  '04': '04 - Fourth Payee Recipient',
  '05': '05 - Fifth Payee Recipient',
  '06': '06 - Sixth Payee Recipient',
  '07': '07 - Seventh Payee Recipient',
  '08': '08 - Eighth Payee Recipient',
  '09': '09 - Ninth Payee Recipient',
  10: '10 - Spouse',
  11: '11 - C&P First Child',
  12: '12 - C&P Second Child',
  13: '13 - C&P Third Child',
  14: '14 - C&P Fourth Child',
  15: '15 - C&P Fifth Child',
  16: '16 - C&P Sixth Child',
  17: '17 - C&P Seventh Child',
  18: '18 - C&P Eighth Child',
  19: '19 - C&P Ninth Child',
  20: '20 - C&P Tenth Child',
  21: '21 - C&P Eleventh Child',
  22: '22 - C&P Twelfth Child',
  23: '23 - C&P Thirteenth Child',
  24: '24 - C&P Fourteenth Child',
  25: '25 - C&P Fifteenth Child',
  26: '26 - C&P Sixteenth Child',
  27: '27 - C&P Seventeenth Child',
  28: '28 - C&P Eighteenth Child',
  29: '29 - C&P Nineteenth Child',
  30: '30 - Vendor',
  31: '31 - Consolidated Payee 1st group of children',
  32: '32 - Consolidated Payee 2nd group of children',
  33: '33 - Consolidated Payee 3rd group of children',
  34: '34 - Consolidated Payee 4th group of children',
  35: '35 - Consolidated Payee 5th group of children',
  36: '36 - Consolidated Payee 6th group of children',
  37: '37 - Consolidated Payee 7th group of children',
  38: '38 - Consolidated Payee 8th group of children',
  39: '39 - Consolidated Payee 9th group of children',
  41: '41 - CH35 First Child',
  42: '42 - CH35 Second Child',
  43: '43 - CH35 Third Child',
  44: '44 - CH35 Fourth Child',
  45: '45 - CH35 Fifth Child',
  46: '46 - CH35 Sixth Child',
  47: '47 - CH35 Seventh Child',
  48: '48 - CH35 Eighth Child',
  49: '49 - CH35 Ninth Child',
  50: '50 - Dependent Father',
  60: '60 - Dependent Mother',
  70: '70 - DIC Award 38 USC412(a) First Payee',
  71: '71 - DIC Award 38 USC412(a) Second Payee',
  72: '72 - DIC Award 38 USC412(a) Third Payee',
  73: '73 - DIC Award 38 USC412(a) Fourth Payee',
  74: '74 - DIC Award 38 USC412(a) Fifth Payee',
  75: '75 - DIC Award 38 USC412(a) Sixth Payee',
  76: '76 - DIC Award 38 USC412(a) Seventh Payee',
  77: '77 - DIC Award 38 USC412(a) Eighth Payee',
  78: '78 - DIC Award 38 USC412(a) Ninth Payee',
  80: '80 - First Claimant Burial/Accrued',
  81: '81 - Second Claimant Burial/Accrued',
  82: '82 - Third Claimant Burial/Accrued',
  83: '83 - Fourth Claimant Burial/Accrued',
  84: '84 - Fifth Claimant Burial/Accrued',
  85: '85 - Sixth Claimant Burial/Accrued',
  86: '86 - Seventh Claimant Burial/Accrued',
  87: '87 - Eighth Claimant Burial/Accrued',
  88: '88 - Ninth Claimant Burial/Accrued',
  89: '89 - Tenth Claimant Burial/Accrued',
  99: '99 - Institutional Veteran CFR3.852'
```

**Comment by alisan16** [comment]
@leikkisa @mbaenatan @nanotone Program Leads had a meeting with John Gosnell and Mike Sanford today where we tried to discuss at a high-level what the Board is expecting from this epic. Some things to note from that discussion:
- Focus on OAR and the Board use cases. No other business lines (i.e. NCA, VHA) for now. If adding support for them is trivial, we can consider it down the line.
- The main Board stakeholders/SMEs have been updated. The Clerk of the Board is the main stakeholder and the Mail Team is also a stakeholder. The Chief of the Clerk of the Board is Donnie Hachey, but he's out for 2 weeks so Sara Cooney can answer questions in his place.
- From what Mike was saying, he was expecting functionality "similar to attorney agent. Can we have a box and enter the name in. These appellants are not in the Corporate DB"
- One thing that was unclear is how VACOLS currently handles non-Veteran appellants, it could be worth reaching out to Jed to see if VACOLS is the source of truth for non-Veteran appellant data on legacy appeals, and whether it is synced with any other VA systems.

Next steps should probably be some requirements gathering with the Board SMEs.
